src/CoreFoundation/Dispatch.cs

```diff
@@ -661,25 +664,24 @@ public DispatchQualityOfService GetQualityOfService (out int relative_priority)
 		// `Equals` and `GetHashCode` based on the Handle property.
 		public override bool Equals (object other)
 		{
-			DispatchQueue o = other as DispatchQueue;
-			if (o == null)
-				return false;
-			return (o.Handle == Handle);
+			if (other is DispatchQueue o)
+				return (o.Handle == Handle);
+			return false;
 		}
 #endif
 
 #if !NET
 		public static bool operator == (DispatchQueue left, DispatchQueue right)
 		{
-			if ((object) left == null)
-				return (object) right == null;
+			if (left as object is null)
+				return right as object is null;
```

Comment:
You don't need the `as object` part:

```suggestion
			if (left is null)
				return right is null;
```